tla-assignments/src/main/scala/at/forsyte/apalache/tla/assignments/SymbTransGenerator.scala

```diff
@@ -247,12 +255,32 @@ class SymbTransGenerator( tracker : TransformationTracker ) {
                         strategy : StrategyType,
                         selections : SelMapType
                       ) : Seq[SymbTrans] =
-      selections( ex.ID ).map( s =>
-        (
-          mkOrdered( s, strategy ),
-          sliceWith( s, selections )( ex )
-        )
-      ).toSeq
+      selections( ex.ID ).map { s =>
+        val ordered = mkOrdered( s, strategy )
+        def rankingFn( tlaEx: TlaEx) : Option[Int] = {
+          // the union of all sets in selections( tlaEx.ID ) lists all the assignments below
```

Comment:
What is it that we are `below` here?

Reply:
Below in the syntax-tree sense, i.e. subexpressions.

Reply:
Ah! How about:

```suggestion
          // the union of all sets in selections( tlaEx.ID ) lists all the assignments in the subexpressions of TlaEx
```